src/main/content/_assets/js/builds.js

```diff
@@ -118,64 +118,86 @@ function render_builds(builds, parent) {
-                                    build.version.lastIndexOf(".") + 1
-                                ),
-                                10
-                            );
+                            
                             if (
+                                buildVersionYear > 21 ||
+                                (buildVersionYear === 21 &&
+                                    buildVersionMonth > 11)
+                            ) {
+                                package_column =
+                                    '<td headers=\'' +
+                                    tableID +
+                                    '_package\'>Jakarta EE 9</td>';
+                            } else if (
                                 buildVersionYear > 19 ||
                                 (buildVersionYear === 19 &&
                                     buildVersionMonth > 5)
                             ) {
-                                var package_column =
-                                    "<td headers='" +
+                                package_column =
+                                    '<td headers=\'' +
                                     tableID +
-                                    "_package'>Jakarta EE 8</td>";
+                                    '_package\'>Jakarta EE 8</td>';
                             } else {
-                                var package_column =
-                                    "<td headers='" +
+                                package_column =
+                                    '<td headers=\'' +
                                     tableID +
-                                    "_package'>Java EE 8</td>";
+                                    '_package\'>Java EE 8</td>';
                             }
-                        } else if (package_name.indexOf("web") > -1) {
-                            var package_column =
-                                "<td headers='" +
+                        } else if (package_name.indexOf('web') > -1) {
+                            var web_profile_version;
+                            if (
+                                buildVersionYear > 21 ||
+                                (buildVersionYear === 21 &&
+                                    buildVersionMonth > 11)
+                            ) {
+                                web_profile_version = 9;
+                            } else {
+                                web_profile_version = 8;
+                            }
+                            package_column =
+                                '<td headers=\'' +
                                 tableID +
-                                "_package'>Web Profile 8</td>";
-                        } else if (package_name.indexOf("microprofile3") > -1) {
-                            var package_column =
+                                '_package\'>Web Profile ' + web_profile_version + '</td>';
```

Comment:
These checks seem too greedy.  Can we keep the old ones as they were (and actually change 'web' to 'webprofile' for the old ones) and then add new ones that specifically look for 'jakartaee9' for 'Jakarta EE 9' and 'webProfile9' for 'Web Profile 9'
Otherwise I'm concerned that some sort of a package with 'java' or 'web' will get added and we'll incorrectly add them to the table.